res_country_state_translations_us/i18n_extra/fr.po

```diff
@@ -0,0 +1,138 @@
+# Translation of Odoo Server.
```

Comment:
@Ehtaga Also there are some states missing.

Reply:
@joaoalf I did not add the states who are the same in English and in French. Only those with a translation are there.